### PR TITLE
EVEREST-420: disable provision command

### PR DESCRIPTION
**Pull request by michal-kralik**
EVEREST-420

### CODE REVIEW

File: commands/root.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func NewRootCmd(l *zap.SugaredLogger) *cobra.Command {
 	rootCmd.PersistentFlags().Bool("json", false, "Set output type to JSON")
 
 	rootCmd.AddCommand(newInstallCmd(l))
-	rootCmd.AddCommand(newProvisionCmd(l))
+	// rootCmd.AddCommand(newProvisionCmd(l))
 	rootCmd.AddCommand(newListCmd(l))
```
Comment (gen1us2k) [inline commands/root.go:41]: Should we disable other commands that are not related to install/uninstall operations?

Comment (michal-kralik) [reply commands/root.go:41]: Since the goal is to get feedback on the functionality, I suggest we keep them.

Comment (recharte) [reply commands/root.go:41]: The other ones are fine but I think the `delete mysql` should also be disabled.
https://github.com/percona/percona-everest-cli/blob/b99b28185e8596fe75925f14f6881b8d09b26ec9/commands/delete.go#L31

Comment (michal-kralik) [reply commands/root.go:41]: Is there any reason you think we shall remove it?

Comment (recharte) [reply commands/root.go:41]: Yes, it's part of the DB management part described in this ticket. I don't see much value in having just the possibility of deleting MySQL clusters. I'd rather not have anything related to this and then deliver it properly.

Comment (michal-kralik) [reply commands/root.go:41]: The ticket states
> Until that is complete we should not release CLI commands that are not working.

The delete command is working and would be great to test in my opinion.
I just removed it though.

**Review by recharte — APPROVED** [review]

**Review by gen1us2k — APPROVED** [review]
